concurrent/actor/ActorPeriodicTask.java

```diff
@@ -39,12 +39,20 @@ public ActorPeriodicTask(Actor<STATE> actor, int timeoutMs, int variationMs, Con
         timer = null;
     }
 
-    public void start() {
+    public void schedule() {
+        schedule(true);
+    }
+
+    public void reschedule() {
         stop();
         schedule(true);
     }
 
-    public void once() {
+    public void scheduleOnce() {
+        schedule(false);
+    }
+
+    public void rescheduleOnce() {
         stop();
         schedule(false);
```

Comment:
What do you think if we get rid of the `schedule(boolean repeat)`? Would we be able to do so without introducing too much duplication?

Reply:
don't forget this @lriuui0x0 ! sorry i approved a tad too early

Reply:
Thinking about it quickly and I didn't think of any. We need to schedule two different versions of lambda in the end.

Do you think of any?

Reply:
okay, let's keep this as is and improve it another time.